### PR TITLE
update README

### PR DESCRIPTION
**Pull request by sallyom**
a few caveats I found when running (and on rhel 9.3)

### CODE REVIEW

**Review by danmcp — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -27,13 +29,21 @@ pip install -r requirements.txt
...
+
+# Now start vllm
 python -m vllm.entrypoints.openai.api_server --model instructlab/granite-7b-lab --tensor-parallel-size 1
 ```
 
 In another shell window
 
 ```shell
 export INSTRUCTLAB_EVAL_FIRST_N_QUESTIONS=10 # Optional if you want to shorten run times
+cd eval
```
[the remark this answers is not in the document]
Comment (sallyom) [reply README.md:46]: sg, i'll update - but when I open a new shell, I'm always dropped into $HOME so I was not in eval in the 2nd shell

File: README.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ Python Library for Evaluation
 
 ## MT-Bench / MT-Bench-Branch Testing Steps
 
+> **⚠️ Note:** Must use Python version 3.11 or later.
```
[the remark this answers is not in the document]
Comment (sallyom) [reply README.md:12]: ok, that I took from a slack thread - but all I know is the default 3.9 from rhel 9.3 is too low. I'll update to 3.10

File: README.md [hunk 3]
```diff
@@ -27,13 +29,21 @@ pip install -r requirements.txt
 pip install -r requirements-dev.txt
 pip install -e .
 pip install vllm
+
+# Ensure the nvidia-toolkit-setup-service is running
+sudo systemctl status nvidia-toolkit-setup.service
+# If it's not running
+sudo systemctl start nvidia-toolkit-setup.service
```
[the remark this answers is not in the document]
Comment (sallyom) [reply README.md:36]: probably not, I opened this b4 #2152 - I'll remove this

[the remark this answers is not in the document]
Comment (sallyom) [reply README.md]: updated

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sallyom) [reply README.md]: updated to comment

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (sallyom) [reply README.md]: done

**Review by danmcp — APPROVED** [review]

**Review by nathan-weinberg — APPROVED** [review]
Thanks Sally!

**Pull request merged**